ansible/roles/eos/tasks/main.yml

```diff
@@ -53,6 +53,8 @@
 - name: build a startup config
   template: src="{{ topo }}-{{ props.swrole }}.j2"
             dest=/mnt/flash/startup-config
-  when: configuration is defined
+  when: hostname in configuration
   notify:
     - Restart the box
+    - Pause for reboot
+    - Wait for VM to come up
```

Comment:
It is dangerous to assume the order of notify item. ref: [https://stackoverflow.com/a/35130254/2514803](https://stackoverflow.com/a/35130254/2514803)

Suggest use normal tasks in this playbook.

Reply:
@qiluo-msft 
Thanks for the suggestion. Updated and committed. Replaced notifying handlers with tasks.